src/common/components/ContactPopup.js

```diff
@@ -72,6 +72,7 @@ const ContactPopup = ({
         } = response;
         const uniqueEmails = [];
         console.log(`Got ${contacts.length} contacts from server`);
+        contacts?.length===0 && setimportContactDialog(true)
```

Comment:
First hide the contact list popup behind the import popup

![hide-contact-list-popup-behind-import-popup](https://user-images.githubusercontent.com/18471335/149874714-62bdc074-52d2-4f9b-ab77-06cee1084dc2.PNG)